src/util/loadWhenDocumentReady.ts

```diff
@@ -0,0 +1,15 @@
+// Utility to load a function when the browsers ready state is complete
+export default function loadWhenReady(fn: () => void): void {
+  if (document.readyState !== "complete") {
```

Comment:
style nit: flipping so truthy condition hits first reads better to me

Reply:
Fixed in: https://github.com/fastly/insights.js/pull/28/commits/2eabbe6176cbf0781efb2ba4c14a0d0ee3f5e8e4